CHANGELOG.md

```diff
@@ -19,7 +19,7 @@ it in future.
 ### Added
 
 ### Changed
-
+* Setting up the Muon shield geometry by ROOT files is completely replaced with the temporary solution of dict in the ```geometry/geometry_config.py```. Set up the shield name is now done by --shieldName key instead of --scName.
```

Comment:
I think I would make an entry for each change (i.e. make a new bullet point after the first sentence.

For readability, I would also use single backticks for the filepath and add a blank line below the heading and after the list.